src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
```

Comment:
Follow the indentation rules. Add an additional space of 2 spaces for nested tags.
https://mate-academy.github.io/style-guides/htmlcss.html#indentation